app/controllers/api/v1/groups_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V1::GroupsController < ApplicationController
+  def index
```

Comment:
This is awesome, but we need a usergroups one too right? The only situation I could see a index of every group being useful is for an admin page. Could be wrong though and maybe thats in the works already! Just wanted to mention it